about.html

```diff
@@ -24,16 +24,16 @@
             <li class="top-nav-item">Logout</li>
         </ul>
         </div>
-        <a href="home.html"><img class="nav-logo" src="img/blockchain-logo.png"></a>
+        <a href="index.html"><img class="nav-logo" src="img/blockchain-logo.png"></a>
```

Comment:
None of your images have alternative text.

<img width="836" alt="Screen Shot 2022-07-27 at 9 29 33" src="https://user-images.githubusercontent.com/82426682/181273490-fcde473f-3b45-48d1-a733-0f17de2774d4.png">

Please add it to all your images.